lib/utils/sendNotifications.js

```diff
@@ -0,0 +1,27 @@
+const users = require('../../lib/modelManagers/followModel');
+const sendVerficationMail = require('../../lib/utils/emailService/emailNotification');
+const articleModelManager = require('../../lib/modelManagers/articlemodel');
+const user = require('../../lib/modelManagers/userProfileModel');
+
+const getUsersToNotify = async (id, title, article) => {
```

Comment:
`getUsersToNotify` does not fully describe what the function is doing. Please renaming the function to properly describe what is going on in the function.

Reply:
I have changed this to sendEmailtoUsers, which describes what the function does.